gradle/verification-metadata.xml

```diff
@@ -281,17 +281,17 @@
    </configuration>
    <components>
       <component group="android-studio" name="android-studio" version="2021.1.1.19">
-         <artifact name="android-studio-2021.1.1.19.windows.zip">
-            <sha256 value="27015ad5e1abb2586de52bc77a0b2b40e902db5dea718a4e2565b2d498e44cbb" origin="Artifact is not signed"/>
+         <artifact name="android-studio-2021.1.1.19.linux.tar.gz">
+            <sha256 value="dc2334357c5026ba7afee7a9ac61c97c127e3d01d1c13e5d125df4df4ac3b4c0" origin="Artifact is not signed"/>
          </artifact>
          <artifact name="android-studio-2021.1.1.19.mac.zip">
             <sha256 value="32a0e81d1de412601ef2d7a1d177d4a5830bb513c46674502bdd833253c69e50" origin="Artifact is not signed"/>
          </artifact>
          <artifact name="android-studio-2021.1.1.19.mac_arm.zip">
             <sha256 value="66e75b427155ba5b07cfa0e7eefc5c3864269411e16674b300a5f475f026a64f" origin="Artifact is not signed"/>
          </artifact>
-         <artifact name="android-studio-2021.1.1.19.linux.tar.gz">
-            <sha256 value="dc2334357c5026ba7afee7a9ac61c97c127e3d01d1c13e5d125df4df4ac3b4c0" origin="Artifact is not signed"/>
+         <artifact name="android-studio-2021.1.1.19.windows.zip">
```

Comment:
It seems like the order of windws/linux artifacts changed. Is this a problem with our code to update the verification metadata or was the order "wrong" before?

Reply:
The order was wrong, the previous edits most likely did not re-run verification metadata generation after doing manual edits.